node/device/samples/simple_sample_device_with_sas.js

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+var Amqp = require('azure-iot-device-amqp').Amqp;
```

Comment:
one thing we have been thinking about when refactoring the other sample is rename all transport vars to `Protocol` and then use `Protocol` in the call on line 19. this limits the number of changes a user has to make to test various transports. again, I'll be happy to do it if you'd rather not spend the time Thanks for the new sample, it's very much appreciated!